Converters/ArcGIS/Speckle.Converters.ArcGIS3/Utils/LocalToGlobalConverterUtils.cs

```diff
@@ -30,100 +31,82 @@ public Base TransformObjects(Base atomicObject, List<Matrix4x4> matrix)
 
     // This is a temp hack. We would add transformations to conversions later instead try to copy objects like this.
     // Jedd also has opinions on this.
-    Base newObject = Core.Api.Operations.Deserialize(Core.Api.Operations.Serialize(atomicObject));
+    // Base newObject = Core.Api.Operations.Deserialize(Core.Api.Operations.Serialize(atomicObject));
+
+    List<Objects.Other.Transform> transforms = new() { };
+    transforms.AddRange(matrix.Select(x => new Objects.Other.Transform(x, "none")).ToList());
 
-    if (newObject.TryGetDisplayValue() is IReadOnlyList<Base> listVals)
+    if (atomicObject is ITransformable c && atomicObject is not SOG.Brep)
     {
-      if (listVals.ToList().FindAll(x => x is not SOG.Mesh).Count == 0)
+      foreach (var transform in transforms)
       {
-        return TransformObjWithDisplayValues(newObject, matrix);
+        c.TransformTo(transform, out ITransformable newObj);
+        c = newObj;
       }
-      throw new SpeckleException(
-        $"Display Values of types '{listVals.ToList().FindAll(x => x is not SOG.Mesh).Select(y => y.speckle_type).Distinct().ToList()}' for {newObject.speckle_type} are not supported for local to global coordinate transformation"
-      );
+
+      atomicObject = (Base)c;
+      foreach (var prop in atomicObject.GetMembers(DynamicBaseMemberType.Dynamic))
+      {
+        atomicObject[prop.Key] = prop.Value;
+      }
+
+      return atomicObject;
     }
-    // TODO: here or preferably in converters elegantly!
-    // else if (newObject is SOG.Polyline polyline)
-    // {
-    //
-    // }
-    // else if (newObject is SOG.Line line)
-    // {
-    //
-    // }
-    // else if (newObject is SOG.Point point)
-    // {
-    //
-    // }
-    if (newObject is SOG.Mesh mesh)
+
+    if (atomicObject.TryGetDisplayValue() is IReadOnlyList<Base> listVals)
```

Comment:
Interesting this Try method doesn't follow the standard C# pattern but instead returns this value if present?